packages/v4/src/content/design-guidelines/components/number input/number-input.md

```diff
@@ -0,0 +1,57 @@
+---
+id: Number input
+section: components
+---
+
+A **number input** provides a quick and effective way for users to enter and modify a numeric value. Similar to <a href="https://www.patternfly.org/v4/components/input-group/">input groups</a>, number inputs combine two input types into one control: A text input field with buttons on either side. When combined, these elements allow users to input numeric values in a responsive and scalable way.
+
+## Elements
+<img src="./img/number_input_anatomy.png" alt="Number input elements" width="537px"/>
+
+1. **Label:** Indicates what value the user should input in the numeric field.
+2. **Input field:** Area for the user to type in their numeric input.
+3. **Buttons:** Increase or decrease the value by set increments.
+4. **Helper text:** Explains or provides additional context that helps the user provide the right information. This is an optional field.
+5. **Unit:** Provides a unit symbol for the value in the input field, when applicable. This is an optional field.
```

Comment:
Could you just add (optional) in brackets right after "Unit"?
so --> 5. Unit (optional):

Reply:
And same thing for helper text! That's usually what we do with optional items in an elements break down!